### PR TITLE
Add routes for separate post page & create form and save button to edit it

### PR DESCRIPTION
**Pull request by solopiy**

### CODE REVIEW

**Comment by solopiy** [comment]
[When I'm on posts page in admin panel and I click on title of post I want to be able to have separate page for the post to change it](https://trello.com/c/ZNxjAfvM/332-when-i-m-on-posts-page-in-admin-panel-and-i-click-on-title-of-post-i-want-to-be-able-to-have-separate-page-for-the-post-to-chang)

**Comment by solopiy** [comment]
[When I'm on posts page in admin panel and I click on title of post I want to be able to have separate page for the post to change it](https://trello.com/c/ZNxjAfvM/332-when-i-m-on-posts-page-in-admin-panel-and-i-click-on-title-of-post-i-want-to-be-able-to-have-separate-page-for-the-post-to-chang)

**Comment by solopiy** [comment]
[When I'm on post page in admin panel and I click on body of post I want to be able to edit it so I can quickly change something](https://trello.com/c/40z9psrD/318-when-i-m-on-post-page-in-admin-panel-and-i-click-on-body-of-post-i-want-to-be-able-to-edit-it-so-i-can-quickly-change-something)

**Comment by solopiy** [comment]
[When I'm on post page in admin panel and I click on body of post I want to be able to edit it so I can quickly change something](https://trello.com/c/40z9psrD/318-when-i-m-on-post-page-in-admin-panel-and-i-click-on-body-of-post-i-want-to-be-able-to-edit-it-so-i-can-quickly-change-something)

File: app/concepts/admin/components/Posts/index.js [hunk 1]
```diff
@@ -2,15 +2,17 @@ import React from 'react';
...
 const Posts = props => (
   <div>
     {
       props.items.length ?
         <ul>
           {props.items.map(item =>
-            <li key={item.id}>
-              <Post object={item} />
-            </li>
+            <Link key={item.id} to={`/admin/posts/${item.id}`} >
```
Comment (Proghat) [inline app/concepts/admin/components/Posts/index.js]: @solopiy, I would like to create a `paths` object where we will store all the internal paths. In order not to duplicate them across the app.

File: app/concepts/admin/components/Post/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import { Link } from 'react-router';
+
+const Post = props => (
+  <div>
+    <form>
+      <input defaultValue={props.posts[0].title}></input>
+      <input defaultValue={props.posts[0].published_at}></input>
+      <input defaultValue={props.posts[0].slug}></input>
+      <button type="submit">Save</button>
+    </form>
```
Comment (ezhlobo) [inline app/concepts/admin/components/Post/index.js]: @Proghat I'm happy with it, but it's still not ready. Let me know if you want me prepare it asap.

File: app/concepts/admin/containers/PostData/actions.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+});
+
+export const receivePost = posts => ({
+  type: RECEIVE_POST,
+  posts,
+});
+
+export const fetchPost = () => (dispatch) => {
+  dispatch(requestPost());
+  return fetch('/cp/posts.json', { credentials: 'include' })
```
Comment (Proghat) [inline app/concepts/admin/containers/PostData/actions.js]: Also, I would store server API paths in separate object too. So we can easily change them in the future.

```js
// app/concepts/admin/server.js

server = {
  routes: {
    posts: () => '/cp/posts.json',
    post: (id) => `/cp/posts/${id}.json`,
  },
};
```

**Comment by Proghat** [comment]
Why have you chosen `PostData`? What do you think about just `Post`?

File: app/concepts/admin/components/Post/index.js (hunk 2 above)
Comment (Proghat) [inline app/concepts/admin/components/Post/index.js]: @ezhlobo, does it cover the case we have here?

**Pull request closed without merging**